src/rpc/client/cdpSession.ts

```diff
@@ -0,0 +1,42 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { CDPSessionChannel, CDPSessionInitializer } from '../channels';
+import { ConnectionScope } from './connection';
+import { ChannelOwner } from './channelOwner';
+import { Protocol } from '../../chromium/protocol';
+
+export class CDPSession extends ChannelOwner<CDPSessionChannel, CDPSessionInitializer> {
+  static from(download: CDPSessionChannel): CDPSession {
+    return (download as any)._object;
```

Comment:
download?

Reply:
Now you know where I've copied this from :)